frontend/include/chpl/types/BoolType.h

```diff
@@ -27,9 +27,7 @@ namespace types {
 
 
 /**
-  This class represents a bool type, e.g. `bool` or `bool(32)`.
-
-  Note that `bool` normally has width `8` but `bool(8)` is a separate type.
+  This class represents a bool type, i.e. `bool`.
```

Comment:
Nit, but "the bool type" or simply "the `bool` type" (without the i.e.) seem clearer now that we don't have variable-width bools.